paasta_tools/mesos/master.py

```diff
@@ -166,12 +166,17 @@ def slaves(self, fltr=""):
             itertools.ifilter(
                 lambda x: fltr == x['id'], self.state['slaves'])))
 
-    def _task_list(self, active_only=False):
+    def _task_list(self, active_only=False, include_orphans=False):
```

Comment:
I'm dubious if this is the right place for this. What do you think about making a new ``get_orphaned_tasks`` function and making the callers be explicit about wanting them?